### DISCUSSION TITLE
Customise domain path

### DISCUSSION THREAD

**Discussion by Ynsnnn** [toeverything/AFFiNE#8428] [Q&A]
This is how I access the application: https://my.domain.com/affine. But it redirects me to https://my.domain.com/js/npm... (for example) and I need https://my.domain.com/affine/js/npm...
I put AFFINE_SERVER_EXTERNAL_URL="https://my.domain.com/affine/" and AFFINE_SERVER_SUB_PATH="/affine/" and it still doesn't work. What env variables should I configure or what am I doing wrong?

**Comment by fawqsir** [comment]
I am running into the same situation.
I have also tried using the AFFINE_SERVER_HOS=my.domain.com with AFFINE_SERVER_SUB_PATH=/affine.
Tried lots of combinations so far and nothing has worked so far. Hoping someone has gotten the Subfolder/Subpath working.

**Comment by Daisuke-sama** [comment]
Hello! Is anybody here? We have been being to the same situation since 11 months

**Comment by ramita-0** [comment]
Please i need an answer too! I centralize my apps under a the same subdomain. So i wont use the subdomain cfg. I need to be able to configure it like this...
